base/markdown/render/plain.jl

```diff
@@ -42,6 +42,8 @@ function plain(io::IO, md::HorizontalRule)
     println(io, "–" ^ 3)
 end
 
+plain(io::IO, md) = writemime(io, "text/plain", md)
```

Comment:
Elsewhere in this file it's referenced as `writemime(io, MIME"text/plain"(), md)`, does this difference not matter?